nhd/NHDScheduler.py

```diff
@@ -58,44 +58,45 @@ def __init__(self, q: Queue):
         self.logger.warning(f'NHD version {self.ver}')
 
 
-    def InitNHDNodes(self):
+    def InitNHDNodes(self,newNodes):
```

Comment:
Please use either 'newnodes' or 'new_nodes' to be consistent with all other function argument names used throughout.

Reply:
Solved